### ISSUE TITLE
can't press `d` to  inspect frame in debugger

### ISSUE THREAD

**Issue by jam1015** [l04m33/vlime#76]
<CR> works though. using SBCL in vim in Alacritty on Arch.

**Comment by phmarek** [comment]
I guess that's because there are other keybindings that start with a `d`. Try `d` and a space, perhaps?

**Comment by jam1015** [comment]
ok there was a plugin that was in conflict; but now instead of opening a new buffer like the tutorial claims, the debuffing info is shown below the frame in the same buffer as the frame list; maybe this is the indended behavior and the tutorial in written imprecisely?

**Comment by phmarek** [comment]
Yeah, well, I guess that the plugin code got touched more recently than the tutorial.

Do you want to help with a pull request?

**Comment by jam1015** [comment]
I could do a pull request to change the tutorial/documentation but I'd want other people to confirm this behavior first.  The reason is I think an actual buffer may have come up once; maybe I'm misremembering or crazy; I don't have time to dig into the actual code to see what it does so I'll hope that other people can confirm or partially - deny my experience before doing anything
